### DISCUSSION TITLE
【HEO主题】关于阅读体验 ，调整行距、字体、添加图片浏览器插件

### DISCUSSION THREAD

**Discussion by jaydong2016** [tangly1024/NotionNext#2093] [Ideas]
### 关于 heo 主题在电脑浏览时的阅读体验
1. 行宽过大，阅读时要转动脖子，降低阅读效率，而且目光从行尾移至下一行首容易串行。 ![Snipaste_2023-08-04_12-07-54.jpg](https://r2.adone.eu.org/2023/a74eec1226132d98a65fe80db9e604f5.jpg)![Snipaste_2023-08-04_13-08-01.png](https://r2.adone.eu.org/2023/bcce95fa148b31cf26820660ce19fa69.png)

2. 页面放大后，字体是变大了，但文字依然充满整个页宽，行宽过大![Snipaste_2023-08-04_12-12-24.jpg](https://r2.adone.eu.org/2023/f1c58e6b8d9ad735c277d439155afc3c.jpg)

3. 右边目录栏建议可以常驻![Snipaste_2023-08-04_12-13-03.jpg](https://r2.adone.eu.org/2023/b97654e494953080c49747a37bb028bf.jpg)

可参考 hexo 和 nobelium 主题的行宽，我觉得是较为适合的，有利于提升阅读体验。

### 关于图片浏览
建议添加一个“图片浏览器”，下图是 notion 分享页面的原生图片浏览功能![Snipaste_2023-08-04_12-23-42.jpg](https://r2.adone.eu.org/2023/5c4ce19893b0a46f4af9197cf8fbd305.jpg)

**Comment by jaydong2016** [comment]
如果能添加一个改变字号大小的按钮也很不错，或者可以考虑适配浏览器自带的阅读模式

**Comment by bbb3n** [comment]
> 还有一点小建议♥，希望heo主题对于移动端的卡片区能更好地适配。卡片展示不完全，右侧会留出空白
> 
> android端截图 ![Screenshot_2023-08-05-00-24-31-84_a252b927494330cdc2c8ba3b3f952e5e.jpg](https://user-images.githubusercontent.com/119606309/258477571-6fe0cc56-df12-4b96-ad84-21708a46ccaa.jpg)
> 
> ipad截图 ![mmexport1691166529757.jpg](https://user-images.githubusercontent.com/119606309/258478430-d162a6c0-c0d1-4c50-8dcf-a5e8e0e4d5d7.jpg)

右边是建议放推荐文章的，使用“推荐”tag

**Comment by happyzhangyyds** [comment]
感谢您的帮助！问题已得到解决♥
